### PR TITLE
fixed some comment errors

### PR DESCRIPTION
**Pull request by BurningCN**
## What is the purpose of the change

fixed some comment errors

## Verifying this change

- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-common/src/main/java/org/apache/dubbo/common/config/Environment.java [hunk 1]
```diff
@@ -131,7 +131,7 @@ public synchronized CompositeConfiguration getPrefixedConfiguration(AbstractConf
             prefixedConfiguration.addConfiguration(propertiesConfiguration);
         } else {
             // The sequence would be: SystemConfiguration -> AbstractConfig -> AppExternalConfiguration -> ExternalConfiguration -> PropertiesConfiguration
-            // Config center has the highest priority
```
Comment (horizonzy) [inline dubbo-common/src/main/java/org/apache/dubbo/common/config/Environment.java:134]: The highest is right, needn't to modify it.

File: dubbo-common/src/main/java/org/apache/dubbo/common/utils/MethodUtils.java [hunk 2]
```diff
@@ -184,7 +184,7 @@ public static boolean isDeprecated(Method method) {
     }
 
     /**
-     * Get all public {@link Method methods} of the declared class, including the inherited methods.
+     * Get all public {@link Method methods} of the declared class, excluding the inherited methods.
```
Comment (horizonzy) [inline dubbo-common/src/main/java/org/apache/dubbo/common/utils/MethodUtils.java:187]: The including is right.

File: dubbo-common/src/main/java/org/apache/dubbo/common/config/Environment.java (hunk 1 above)
Comment (BurningCN) [reply dubbo-common/src/main/java/org/apache/dubbo/common/config/Environment.java:134]: OK, according to your suggestion, I will modify this problem, should I directly continue to modify under this PR (which will result in multiple commit logs), or launch a new PR to modify (this one will only have one commit log)?

Comment (horizonzy) [reply dubbo-common/src/main/java/org/apache/dubbo/common/config/Environment.java:134]: commit and push directly.

Comment (BurningCN) [reply dubbo-common/src/main/java/org/apache/dubbo/common/config/Environment.java]: ok, I have finished the modification

Comment (horizonzy) [reply dubbo-common/src/main/java/org/apache/dubbo/common/config/Environment.java]: Other also need hanlde.

**Pull request closed without merging**